src/fsharp/CompileOptions.fs

```diff
@@ -172,6 +172,32 @@ let DumpCompilerOptionBlocks blocks = List.iter dumpCompilerOptionBlock blocks
 let isSlashOpt (opt:string) = 
     opt.[0] = '/' && (opt.Length = 1 || not (opt.[1..].Contains "/"))
 
+module ResponseFile =
+
+    type ResponseFileData = ResponseFileLine list
+    and ResponseFileLine =
+        | CompilerOptionSpec of string
+        | Comment of string
+
+    let parseFile path : Choice<ResponseFileData,Exception> =
+        let parseLine (l: string) =
+            match l with
+            | s when String.IsNullOrWhiteSpace(s) -> None
+            | s when l.StartsWith("#") -> Some (ResponseFileLine.Comment (s.TrimStart('#')))
+            | s -> Some (ResponseFileLine.CompilerOptionSpec (s.Trim()))
+
+        try
+            use stream = FileSystem.FileStreamReadShim path
+            use reader = new System.IO.StreamReader(stream, System.Text.Encoding.UTF8)
```

Comment:
Why not to do it exactly like roslyn in [here](https://github.com/dotnet/roslyn/blob/3c51bcad7bacb76ee300d9c48d67888d6057955f/src/Compilers/Core/Portable/CommandLine/CommonCommandLineParser.cs#L39-L44)